_guides/dhis2-app-checklist.md

```diff
@@ -0,0 +1,37 @@
+## Starting a new app
+
+### Bootstrap
+
+-   [][create react app](https://github.com/facebook/create-react-app)
+
+## Code style
+
+-   [][dhis2/code-style](https://github.com/dhis2/code-style)
+
+## Component library
+
+-   [][dhis2/ui](https://github.com/dhis2/ui)
+
+## Documentation
+
+-   [][jsdoc3](https://github.com/jsdoc3/jsdoc)
+
+## Tests
+
+-   [] Unit
+-   [] Integration
+-   [] End to End
+
+## Tools
+
+-   [][dhis2/packages](https://github.com/dhis2/packages)
```

Comment:
Should we remove `packages`, `deploy-build`, and `fossa` since they are deprecated and not really recommended for new app projects?

Reply:
Entire file can be removed (at least the content). I ran the formatter on it, that's why it shows up.